.github/workflows/gradle.yml

```
@@ -1,44 +1,83 @@
 name: Deploy to EC2
 
 on:
-  push:
-    branches: [ dev ]  # dev 브랜치에 push 될 때 실행
+  pull_request:
+    branches: [ dev ]  # dev 브랜치에 PR 발생 시 실행
+    types:
+      - opend
```

actionlint (1.7.7)

invalid activity type "opend" for "pull_request" Webhook event. available types are "assigned", "auto_merge_disabled", "auto_merge_enabled", "closed", "converted_to_draft", "demilestoned", "dequeued", "edited", "enqueued", "labeled", "locked", "milestoned", "opened", "ready_for_review", "reopened", "review_request_removed", "review_requested", "synchronize", "unassigned", "unlabeled", "unlocked"

(events)